packages/core-contracts/contracts/mocks/utils/OwnableMock.sol

```diff
@@ -0,0 +1,37 @@
+//SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "../../utils/Ownable.sol";
+
+contract OwnableMock is Ownable {
+    address public override owner;
+    address public override nominatedOwner;
+
+    constructor(address firstOwner) {
+        require(firstOwner != address(0), "Owner cannot be 0x0");
+        owner = firstOwner;
+        emit OwnerChanged(address(0), firstOwner);
+    }
+
+    function nominateNewOwner(address ownerNominated) external override onlyOwner {
```

Comment:
These functions shouldn't be implemented in the mock, but in Ownable.sol.

The only abstract __internal methods in Ownable.sol are `_setOwner`, _getOwner`, `_setNominatedOwner`, and _getNominatedOwner`.

So Ownable defines the functionality but does not specify where things are stored. The mock overrides the functions above to store these in regular storage.